src/textual/geometry.py

```diff
@@ -352,7 +401,7 @@ def clip(self, width: int, height: int) -> Region:
         return new_region
 
     def intersection(self, region: Region) -> Region:
-        """Get that covers both regions.
+        """Return a new instance describing the intersection between two regions.
```

Comment:
I messed up the original, but the change doesn't suggest what intersection means.